models/modules/unet_generator_attn/unet_attn_utils.py

```diff
@@ -8,6 +8,35 @@
 import torch.nn as nn
 from einops import rearrange
 from .switchable_norm import SwitchNorm2d
+from einops import rearrange
+
+
+class InflatedConv3d(nn.Conv2d):
```

Comment:
This class seems to already be in this file.

Reply:
repreated classes "InflatedConv3d" and "InflatedGroupNorm" are delected